pkg/globalnet/controllers/ipam/globalnet_ingress.go

```diff
@@ -60,3 +60,22 @@ func (i *Controller) doesIPTablesChainExist(table, chain string) (bool, error) {
 
 	return false, nil
 }
+
+func (i *Controller) updateIngressRulesForHealthCheck(resourceName, cniIfaceIP, globalIP string, addRules bool) error {
+	ruleSpec := []string{"-p", "icmp", "-d", globalIP, "-j", "DNAT", "--to", cniIfaceIP}
```

Comment:
I'd remove the specific icmp protocol, and leave that free. For example it could be helpful to be able to debug over TCP/UDP if necessary.

Reply:
Thanks for reviewing the PR @mangelajo 

Can we do any such change as a separate PR as this PR is to support HealthCheck which uses icmp.